src/sail_json_backend/json.ml

```diff
@@ -147,6 +148,7 @@ let rec string_list_of_mpat x = match x with
       List.concat (List.map string_list_of_mpat mpl)
   | _ -> assert false
 
+
```

Comment:
Minimize extraneous changes, especially in whitespace. It makes the code easier to review if the only changes are code changes. Here, a blank line was added.

Reply:
Oh no! I did not see the whitespaces adding up like that. I squashed a bunch of commits and they must've added up! Sorry, fixing now!